src/combinatorics/simple_graph/connectivity.lean

```diff
@@ -1620,6 +1631,26 @@ begin
     apply set.to_finite, },
 end
 
+lemma to_subgraph_map_hom_le {H : G.subgraph} {u v : H.verts} (p : H.coe.walk u v) :
+  subgraph.map H.hom p.to_subgraph ≤ H :=
+begin
+  split,
+  { simp only [subgraph.map_verts, subgraph.hom_apply, set.image_subset_iff,
+               subtype.coe_preimage_self, set.subset_univ], },
+  { rintro x y ⟨_, _, a, rfl, rfl⟩,
+    exact p.to_subgraph.adj_sub a, },
+end
```

Comment:
There is a much more general lemma here:
```lean
lemma subgraph.coe_subgraph_le {H : G.subgraph} (H' : H.coe.subgraph) :
  H'.coe_subgraph ≤ H :=
begin
  split,
  { simp only [subgraph.map_verts, subgraph.hom_apply, set.image_subset_iff,
      subtype.coe_preimage_self, set.subset_univ] },
  { rintro v w ⟨_, _, h, rfl, rfl⟩,
    exact H'.adj_sub h }
end
```

Then you don't need `to_subgraph_map_hom_le` since it proof is
```lean
lemma to_subgraph_map_hom_le {H : G.subgraph} {u v : H.verts} (p : H.coe.walk u v) :
  subgraph.map H.hom p.to_subgraph ≤ H :=
subgraph.coe_subgraph_le _
```